src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -132,6 +132,7 @@ public void CB_START_BARRACK(LoginConnection conn, Packet packet)
 			Send.BC_SERVER_ENTRY(conn, "127.0.0.1", 9001, "127.0.0.1", 9002);
```

Comment:
Unrelated to this PR, these should be migrated to settings. Additionally, it is completely fine to send just one set of IP address and port.

Reply:
That was... the social server? Mhm, you're right, I guess we just hardcoded it because the social server doesn't exist in Melia yet.